### PR TITLE
chore: add accessibility ids

### PR DESCRIPTION
**Pull request by Taxxodium**
## Description

- added accessibility ids
- version bump

## Known issues

Restart, Play and Seek buttons have no accessibility id because we can't access these buttons from code. JWPlayer does not make these buttons public.

## Checklist

* [X] PR is scoped to one task
* [ ] Tests are included
* [ ] Documentation is included

* [X] This PR is not making any UI change
* [ ] This PR is making a UI change
  * [ ] Screenshot(s) included

## QA :

* required test cases:
* specific apps / plugins to test :

### CODE REVIEW

File: iOS/JWPlayerPlugin/JWPlayerViewController.m [hunk 1]
```diff
@@ -87,6 +98,7 @@ - (UIButton *)closeButton {
     button.titleLabel.font = [UIFont systemFontOfSize:25];
     [button setTitle:@"X" forState:UIControlStateNormal];
     [button setTitleColor:UIColor.whiteColor forState:UIControlStateNormal];
+    [button setAccessibilityIdentifier:kJWPlayerCloseButton];
```
Comment (mncws) [inline iOS/JWPlayerPlugin/JWPlayerViewController.m]: Views\controls may not be accessible via Xcode Accessibility Inspector or other related software if view\control never passed this method: `[view setIsAccessibilityElement:YES];`

Comment (Taxxodium) [reply iOS/JWPlayerPlugin/JWPlayerViewController.m]: According to Apple's documentation in regards to this method: 

> The default value for this property is NO unless the receiver is a standard UIKit control, in which case the value is YES.

In this case we are dealing with standard UIKit controls (UIButtons and UIViews), so this button should be accessible via the Accessibility Inspector.

**Review by kononenkoAnton — APPROVED** [review]

File: Specs/JWPlayerPlugin/3.3.4/JWPlayerPlugin.podspec [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+
+  s.name             = "JWPlayerPlugin"
+  s.version          = '3.3.4'
+  s.summary          = 'JWPlayer Player plugin implementation.'
+  s.description      = 'An implementation for JWPlayer as a Zapp Player Plugin in Objective C.'
+  s.homepage         = "https://github.com/applicaster/zapp-player-plugin-jw"
+  s.license          = 'MIT'
+  s.author           = { "Jesus De Meyer" => "j.demeyer@applicaster.com" }
+  s.source           = { :git => "git@github.com:applicaster/zapp-player-plugin-jw.git", :tag => s.version.to_s }
```
Comment (rkarpievich-sc) [inline Specs/JWPlayerPlugin/3.3.4/JWPlayerPlugin.podspec]: We use another name conventions for tag names in monorepos. All new tags should have format like "ios-1.1.1" that also imply changes in pod specs. You can look at [cleeng repo](https://github.com/applicaster/zapp-login-plugin-cleeng/blob/master/Specs/CleengLogin/1.1.0/CleengLogin.podspec) as reference

Comment (Taxxodium) [reply Specs/JWPlayerPlugin/3.3.4/JWPlayerPlugin.podspec]: PR updated :)

**Review by rkarpievich-sc — APPROVED** [review]

**Pull request merged**